cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -686,44 +724,6 @@ def test_snapshot_tree_view_functionality():
     pass
 
 
-@pytest.mark.manual
-@pytest.mark.provider([VMwareProvider])
-@pytest.mark.tier(1)
-@pytest.mark.meta(coverage=[1395116])
-def test_snapshot_link_after_deleting_snapshot():
-    """
-    test snapshot link in vm summary page after deleting snapshot
-    Have a vm, create couple of snapshots. Delete one snapshot. From the
-    vm summary page use the history button and try to go back to
-    snapshots. Go to the vm summary page again and try to click snapshots
-    link, it should work.
-
-    Polarion:
-        assignee: prichard
-        casecomponent: Infra
-        caseimportance: medium
-        initialEstimate: 1/6h
-        setup:
-            1. Add vmware provider
-            2. Create testing VM
-        testSteps:
```

Comment:
optional: For automated test `testSetps` not required but if available I would do copy paste.

Reply:
Added.